.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
+    "plugins": ["prettier-plugin-solidity"],
+    "overrides": [
+        {
+            "files": "*.sol",
+            "options": {
+                "parser": "solidity-parse"
+            }
+        }
+    ],
+    "tabWidth": 2,
+    "printWidth": 80,
+    "trailingComma": "all",
```

Comment:
I might be inclined to drop this, it seems to mostly add cruft to our javascript